instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/internal/BedrockRuntimeAccess.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awssdk.v2_2.internal;
+
+import io.opentelemetry.javaagent.tooling.muzzle.NoMuzzle;
+import java.util.List;
+import javax.annotation.Nullable;
+import software.amazon.awssdk.core.SdkRequest;
+import software.amazon.awssdk.core.SdkResponse;
+
+final class BedrockRuntimeAccess {
+  private BedrockRuntimeAccess() {}
+
+  private static final boolean enabled;
+
+  static {
+    boolean isEnabled = true;
+    if (!PluginImplUtil.isImplPresent("BedrockRuntimeImpl")) {
+      // Muzzle disabled the instrumentation.
+      isEnabled = false;
+    } else {
+      try {
+        Class.forName("software.amazon.awssdk.services.bedrockruntime.model.ConverseRequest");
```

Comment:
I think unlike SQS / SNS, because there is no entrypoint added for wrapping, I need this for safety when bedrock isn't included (like in `:test` target)